commands/message.go

```diff
@@ -60,9 +60,9 @@ var msgSendCmd = &cmds.Command{
 			return err
 		}
 
-		val, ok := req.Options["value"].(int)
+		val, ok := types.NewAttoFILFromFILString(req.Options["value"].(string))
 		if !ok {
-			val = 0
+			val = types.NewZeroAttoFIL()
```

Comment:
I think it makes sense to actually return an error in this case. Decimal FIL amounts can be malformed in a bunch of ways, and having the user send a message whose value is `0` seems bad.